doozerlib/olm/bundle.py

```diff
@@ -560,9 +560,18 @@ def redhat_delivery_tags(self):
 
     @property
     def operator_framework_tags(self):
+        override_channel = self.channel
+        override_default = self.channel
+        stable_channel = 'stable'
+
+        # see: issues.redhat.com/browse/ART-3107
+        if self.runtime.group_config.operator_channel_stable:
+            override_channel = ','.join([self.channel, stable_channel])
```

Comment:
```suggestion
            override_channel = ','.join({self.channel, stable_channel})
```
... would ensure that they're unique, right?
i want the owners to switch their operator channels to `stable` in the source at least at next branch cut if not before.

Reply:
+1